modules/azure/src/test/java/org/testcontainers/containers/CosmosDBEmulatorContainerTest.java

```diff
@@ -20,13 +24,23 @@
     // }
 
     @Test
-    public void testSimple() {
+    public void testSimple() throws Exception {
+    // buildAndSaveNewKeyStore {
+        Path keyStoreFile = Files.createTempFile("emulator", ".keystore");
+        KeyStore keyStore = emulator.buildNewKeyStore("/tmp/cosmos/appdata/default.sslcert.pfx");
+        keyStore.store(new FileOutputStream(keyStoreFile.toFile()), emulator.getEmulatorKey().toCharArray());
+    // }
+    // setSystemTrustStoreParameters {
+        System.setProperty("javax.net.ssl.trustStore", keyStoreFile.toString());
+        System.setProperty("javax.net.ssl.trustStorePassword", emulator.getEmulatorKey());
+        System.setProperty("javax.net.ssl.trustStoreType", "PKCS12");
+    // }
```

Comment:
Can we move these to CosmosEmulator with an explicit call? 
/cc @kiview

Reply:
I would rather not, see our previous discussion on explicitly this.

Reply:
@cemo this hopefully will be addressed on Azure SDK's side - not something we can/should do about it